yield_plugin/include/yield_plugin/yield_plugin.hpp

```diff
@@ -221,8 +220,22 @@ class YieldPlugin
    * \brief Callback for map projection string to define lat/lon -> map conversion
    * \brief msg The proj string defining the projection.
    */ 
-  void georeferenceCallback(const std_msgs::msg::String::UniquePtr msg);
+  void georeference_callback(const std_msgs::msg::String::UniquePtr msg);
+  
+  /**
+   * \brief Callback for external objects with predictions in the environment
+   * \param msg The object list msg.
+   */ 
+  void external_objects_callback(const carma_perception_msgs::msg::ExternalObjectList::UniquePtr msg);
```

Comment:
Why are we using `std::unique_ptr` here? As far as I know, ROS does not give exclusive ownership to the received messages. If we are making a `std::unique_ptr` from a `std::shared_ptr`, I believe we are making a copy of the underlying message.